File: VersionJava/Pedido.java
public class Pedido {
    private final int id;
    private final String nombre;

    public Pedido(int id) {
        this.id = id;
        this.nombre = "Pedido-" + id;
    }

    public int getId() {
        return id;
    }

    public String getNombre() {
        return nombre;
    }
}

File: version_python/pedido.py
class Pedido:
    """
    Representa un pedido en la cocina.
    Tiene un ID y el nombre del plato.
    """

    def __init__(self, id_pedido: int, nombre_plato: str):
        self.id = id_pedido
        self.plato = nombre_plato

    def __str__(self):
        return f"Pedido {self.id}: {self.plato}"
